docs/configuration/logging.md

```diff
@@ -107,8 +107,9 @@ The following example log4j2.xml is based upon the micro quickstart:
 
 > NOTE:
 > Although Druid shares the log4j configuration file task peon processes,
-> the appenders in this file DO NOT take effect for peon processes. Peons always output logs to standard output.
-> Middle Managers redirect task logs from standard output to [long-term storage](index.md#log-long-term-storage).
+> the appenders in this file WILL NOT take effect for peon processes if there is no console appender defined.
+> Peons always output logs to standard output. Middle Managers redirect task logs from standard output to 
+> [long-term storage](index.md#log-long-term-storage).
```

Comment:
This information seems more of a statement of fact; consider moving outside the note block. I think it also interrupts the idea flow to log-level settings in the following lines.